lightningd-alice/entrypoint.sh

```diff
@@ -7,6 +7,10 @@ echo "Core-Lightning starting"
 while read -r i; do if [ "$i" = "lightning-rpc" ]; then break; fi; done \
     < <(inotifywait -e create,open --format '%f' --quiet "/data/.lightning/regtest" --monitor)
 
+sleep 2 # Give Core-Lightning some time to start and generate keys and certificates.
+echo "Changing permissions for /data/.lightning/regtest/client-key.pem"
```

Comment:
let's try not to sleep, but do the same thing as above. Wait for the file to exist and then change its permissions.